src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs

```diff
@@ -22,7 +22,7 @@ namespace Serilog.Enrichers
     /// </summary>
     public class ProcessIdEnricher : ILogEventEnricher
     {
-        LogEventProperty _cachedProperty;
+        private LogEventProperty? _cachedProperty;
```

Comment:
Nitpick; the `serilog/` org projects don't use explicit `private` or `internal`

Reply:
Ok.
Actually, that makes me wonder if we should import the .editorconfig out of the main repo to help make the settings consistent (the one here is basically empty)